mmcv/ops/csrc/pytorch/bias_act.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+//
+// NVIDIA CORPORATION and its licensors retain all intellectual property
+// and proprietary rights in and to this software, related documentation
+// and any modifications thereto.  Any use, reproduction, disclosure or
+// distribution of this software and related documentation without an express
+// license agreement from NVIDIA CORPORATION is strictly prohibited.
+
+//------------------------------------------------------------------------
+// CUDA kernel parameters.
+
+#ifndef BIAS_ACT_H
+#define BIAS_ACT_H
+
+struct bias_act_kernel_params {
```

Comment:
As this head is only included in bias_act_cuda.cu, can we just move this struct into bias_act_cuda.cu and remove this head?

Reply:
Same for `filtered_lrelu.h` and `upfirdn2d.h`.